### PR TITLE
[DOCFIX] Update docs for setting up configuration, with bootstrap-conf

### PR DESCRIPTION
**Pull request by ifcharming**

### CODE REVIEW

**Comment by ifcharming** [comment]
@apc999

**Comment by apc999** [comment]
+1 LGTM

**Comment by ifcharming** [comment]
@jsimsa  can you also take a look? thanks

File: docs/en/Configuring-Alluxio-with-GCS.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ Then, if you haven't already done so, create your configuration file from the te
 
 {% include Common-Commands/copy-alluxio-env.md %}
```
Comment (jsimsa) [inline docs/en/Configuring-Alluxio-with-GCS.md]: this is no longer accurate, the bootstrap-conf should be the recommended mechanism ... the alternative should be that users copy the `alluxio-env.sh.template` and set the contents manually

**Comment by jsimsa** [comment]
thanks, left one comment

File: docs/en/Configuring-Alluxio-with-GCS.md (hunk 1 above)
Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-GCS.md]: Updated, thanks.

**Comment by ifcharming** [comment]
thanks @jsimsa , updated and made `bootstrap-conf` recommended way, PTAL

**Comment by jsimsa** [comment]
looks great, thank you

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**